src/components/PostList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PostHandler from './PostHandler';
+
+function PostList(props) {
+  if (!props.requested) {
+    return (
+      <div className="initial-button-container">
+        <button  title="click to load posts" className="initial-button" type="button" onClick={props.buttonClick}>Load</button>
+      </div>
+    );
+  }
+  if (props.data === null) {
+    return (
+      <div className="preloader">
+        <div className="preloader-title">Loading...</div>
+        <div className="lds-dual-ring"></div>
+      </div>
+    );
+  }
+  return (
+    <>
+    <input type="text" placeholder="Search" onChange={(event) => props.getInputValue(event)}/>
+    <div className="posts">
+      {
+        props.search ?
```

Comment:
format the code
https://github.com/mate-academy/style-guides/blob/master/javascript.md#comparison-operators--equality